poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/OPCPackage.java

```diff
@@ -141,14 +141,26 @@ public abstract class OPCPackage implements RelationshipSource, Closeable {
      * @throws OpenXML4JRuntimeException if there are issues creating properties part
      */
     OPCPackage(PackageAccess access) {
+        this(access, OPCComplianceFlags.enforceAll());
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param access Package access.
+     * @param opcComplianceFlags Enable or disable specific OPC compliance flags.
+     *                           This is useful to allow parsing of certain non-compliant documents.
+     * @throws OpenXML4JRuntimeException if there are issues creating properties part
+     */
+    OPCPackage(PackageAccess access, OPCComplianceFlags opcComplianceFlags) {
```

Comment:
can you add `@since POI 5.4.1` to new non-private methods and constructors?

Reply:
it's ok with new classes just to put this `@since` on the class level content and not on methods/constructors.